test/compflags/checks/checks.compopts

```diff
@@ -0,0 +1,7 @@
+--checks # checks.good
+--no-checks # checks-nochecks.good
+--fast # checks-fast.good
+--checks --no-checks # checks-checks-nochecks.good
+--no-checks --checks # checks-nochecks-checks.good
+--checks --fast # checks-checks-fast.good
+--no-checks --fast # checks-nochecks-fast.good
```

Comment:
Since there are only two possible outputs in all these configurations (right?), let's just create two .good files (calling them `checks.good` and `checks-off.good`, say) and then have the various compopts cases map to one of those two cases rather than having a .good file per combination of flags.

Also, I can never recall whether a compopts file whose last line doesn't have a linefeed will work or be ignored, so to be safe, let's add a linefeed to the end of this file.

Reply:
I created more files because I thought the expected behavior could change in the future. For example, someday `--fast` won't imply `--no-checks`. Anyway, if that does happen, we can change tests then. I have followed your advice.